### DISCUSSION TITLE
Are Wiki pages lost after enterprise plan is finished?

### DISCUSSION THREAD

**Discussion by ancago** [community/community#49408] [Enterprise]
### Select Topic Area

Question

### Body

When the enterprise plan ends you loose the access to the Wiki pages. Is the access recovered to all the documents in Wiki after you upgrade your plan again? Or will the documents be lost if the plan wasn't continued for a while?

**Comment by Anmol-Baranwal** [comment]
- Wikis are available in `public` repositories with GitHub Free and GitHub Free for organizations.

- Wikis are available in `public and private` repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud and GitHub Enterprise Server.

So, are you asking for a private repository?

**Reply by ancago** [reply]
yes, it's a private repository.
It was Enterprise by 04.03.2023 and after that day it downgraded and I haveno access to Wiki. Will I get it back (with my docs) if I upgrade again?

**Reply by Anmol-Baranwal** [reply]
> I don't know for sure. Some sources say that you will get it back.

But I strongly recommend you contact support [here](https://support.github.com/request)

You will get a satisfactory response.

**Reply by ancago** [reply]
ok thank you!

**Reply by Anmol-Baranwal** [reply]
@ancago 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking any of the two answer as accepted? (More helpful one)

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by 0xNevo** [comment]
If your enterprise plan ends and you lose access to Wiki pages, upgrading your plan again will restore access to the documents in the Wiki. Generally, all the documents will be saved and can be accessed once you upgrade your plan again.

However, it's essential to note that some features and functionality of Wiki may vary based on your plan. If your previous subscription had a feature that is not present in your new plan, you may only recover access to the content specific to your current subscription.

Moreover, it's always best practice to maintain a backup of critical documents by exporting or downloading them before the plan ends. In case there are any issues during the upgrade or after the restoration of the account, you can use these backups to avoid losing the information in the exported files.

**Reply by ancago** [reply]
great, thank you for your help!
